optuna_dashboard/ts/components/StudyDetail.tsx

```diff
@@ -159,14 +159,14 @@ export const StudyDetail: FC<{
           >
             {title}
           </Typography>
-          {graphVisibility.history ? (
+          {studyDetail !== null && graphVisibility.history ? (
             <Card
               sx={{
                 margin: theme.spacing(2),
               }}
             >
               <CardContent>
-                <GraphHistory study={studyDetail} />
+                <GraphHistory studies={[studyDetail]} />
```

Comment:
How about passing an empty list and display an empty graph? It can increase the CLS score.